### ISSUE TITLE
Add support at the chain_id generator command for Gnosis 

### ISSUE THREAD

**Issue by cortze** [probe-lab/hermes#16]
# Description
We are currently supporting all the Etherum main networks:
- Mainnet
- Holesky
- Sepolia
- Prater (although is deprecated)

However, there is no data yet for the Gnosis chain (which shares all the internals with Ethereum)

# Tasks
- [ ] Add the Gnosis network support at the `chain-id`s command

**Comment by namn-grg** [comment]
> Gnosis Chain doesn't support Prysm at the moment.
Source: https://docs.gnosischain.com/node/

Hermes requires/supports Prysm only, right?

**Comment by cortze** [comment]
Yes, Hermes needs a functional Prysm client for the full operation.

We should still be able to get the genesis and chain params to compose the `chain-ids` / `fork_digests` of the network though at the `chain_ids` command tough. 

Thanks for the pointer @namn-grg !

**Comment by cortze** [comment]
closes #52
